examples/opmpack.cpp

```diff
@@ -75,8 +75,12 @@ but restart and import files referred to in the deck are also copied. The -o and
 
 
 void copy_file(const fs::path& source_dir, fs::path fname, const fs::path& target_dir) {
-    if (fname.is_absolute())
-        fname = fs::relative(fname, source_dir);
+    if (fname.is_absolute()) {
```

Comment:
Thanks for fixing. Could maybe leave the `fs::relative()` alternative as a comment?

Reply:
okay, will do.